docs/seeded.md

```diff
@@ -0,0 +1,59 @@
+# Seeded Topic Modeling
+
+When investigating a set of documents, you might already have an idea about what aspects you would like to explore.
+Some models are able to account for this by taking seed phrases or words.
+This is currently only possible with KeyNMF in Turftopic, but will likely be extended in the future.
+
+In [KeyNMF](../keynmf.md), you can describe the aspect, from which you want to investigate your corpus, using a free-text seed-phrase,
```

Comment:
See the comment above as well.

Would probably write this more simply and then put this in a Tab section called KeyNMF (that way it is easy to see that the only one supported in KeyNMF, but also that there could be others in the future.